src/main/java/life/qbic/expdesign/io/MHCLigandDesignReader.java

```diff
@@ -90,7 +91,7 @@ public String[] getMHCClass(String antibody) {
     if (antibodyToMHCClass.containsKey(antibody))
       return antibodyToMHCClass.get(antibody);
     else {
-      logger.error(antibody + " is an unknown antibody. Please make sure to only use pre-defined antibodies.");
+      error = antibody + " is an unknown antibody. Please make sure to only use pre-defined antibodies.";
```

Comment:
If i understood the error handling in this class correctly: 
1.) The error message is set explicitly to a string
2.) The class returns null 
3.) The execution chain is continued if it's not null
4.) At the end of the class log error string if not empty 
I'm curious why we don't keep using the logger that's already implemented?

Reply:
The project wizard fetches the string to present users with the specified error. Nowadays I would probably do that with exceptions.